### PR TITLE
A4A fetch include option requestAmpResponseSourceOrigin

### PR DESCRIPTION
**Pull request by keithwrightbos**

### CODE REVIEW

File: extensions/amp-a4a/0.1/amp-a4a.js [hunk 1]
```diff
@@ -483,8 +483,7 @@ export class AmpA4A extends AMP.BaseElement {
       mode: 'cors',
       method: 'GET',
       credentials: 'include',
-      // TODO(kjwright):  Add requireAmpResponseSourceOrigin once supported
-      // server-side
+      requireAmpResponseSourceOrigin: true
```
Comment (jridgewell) [inline extensions/amp-a4a/0.1/amp-a4a.js]: Need a trailing comma

File: extensions/amp-a4a/0.1/test/test-amp-a4a.js [hunk 2]
```diff
@@ -118,6 +118,7 @@ describe('amp-a4a', () => {
         mode: 'cors',
         method: 'GET',
         credentials: 'include',
+        requireAmpResponseSourceOrigin: true
```
Comment (jridgewell) [inline extensions/amp-a4a/0.1/test/test-amp-a4a.js]: ditto comma.

File: extensions/amp-a4a/0.1/amp-a4a.js (hunk 1 above)
Comment (keithwrightbos) [inline extensions/amp-a4a/0.1/amp-a4a.js]: Done

File: extensions/amp-a4a/0.1/test/test-amp-a4a.js (hunk 2 above)
Comment (keithwrightbos) [inline extensions/amp-a4a/0.1/test/test-amp-a4a.js]: Done

**Comment by keithwrightbos** [comment]
@jridgewell comments addressed.  Thanks!

**Comment by lannka** [comment]
LGTM

**Pull request merged**